02-peer-protocol.md

```diff
@@ -460,25 +466,61 @@ is negotiated.
 
 This message indicates that the funding transaction has reached the `minimum_depth` asked for in `accept_channel`. Once both nodes have sent this, the channel enters normal operating mode.
 
+Note that `minimum_depth` can be 0 if the accepter trusts the opener,
+in which case this message is a misnomer.  The opener is free to send this
+message at any time (since it presumably trusts itself).
+
 1. type: 36 (`funding_locked`)
 2. data:
     * [`channel_id`:`channel_id`]
     * [`point`:`next_per_commitment_point`]
```

Comment:
Shall we just call it `channel_ready`, since that's the semantics were trying to convey? Fwiw I'm also happy to just keep the current name and remove the misnomer comment (specs are authoritative, and not intended to convey all the details and rationale since they tend to bloat and obfuscate the important Infos).

Reply:
Oh, I meant next_per_commitment_point specifically, not `funding_locked`. I'm happy to see `funding_locked` renamed `channel_ready`, but it'd be nice to see `next_per_commitment_point` renamed as well given it too is a misnomer.

Reply:
It's actually the *second* per-commitment point though.  (We already have first_per_commitment_point in `open_channel`).  But renaming is easy; will check that the `next_per_commitment_point` requirements elsewhere apply properly...